### PR TITLE
Page/add topic

### PR DESCRIPTION
**Pull request by F-Kirchhoff**
- add a Add Topic route and page
- adapt AddTopic Form for new environment

### CODE REVIEW

File: src/app/components/OverlayWrapper/OverlayWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  bottom: 0;
+  right: 0;
+`
+const FormBackground = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background-color: rgba(0, 0, 0, 0.7);
```
Comment (julialoeschel) [inline src/app/components/OverlayWrapper/OverlayWrapper.tsx:34]: why is this new, I just reviewed this one PR earlier :)

File: src/app/App.tsx [hunk 2]
```diff
@@ -136,20 +132,50 @@ function App(): JSX.Element {
...
+  const params = useParams()
+  const topic = topics.find(topic => topic.id === params.topicId)
+  return topic ? (
+    <TopicView
+      onUpvoteChange={onNeedUpvote(topic.id)}
+      onNeedSubmit={onNeedSubmit(topic.id)}
+      content={topic}
+    />
+  ) : (
+    <h1>404 Page not found</h1>
```
Comment (julialoeschel) [inline src/app/App.tsx:179]: Same code as in PR#31?

**Review by julialoeschel — COMMENTED** [review]
great! im confused why i see the same exact (i think) changes in 2 PRs, but other than that, everything nice!

File: src/app/App.tsx (hunk 2 above)
Comment (F-Kirchhoff) [reply src/app/App.tsx:179]: ups my bad, I guess its because I created this branch from the last page branch I used for the previous PR.

**Pull request merged**